### PR TITLE
[DON'T MERGE] Category style for ReviewApp

### PR DESCRIPTION
**Pull request by okkez**
For https://github.com/bm-sms/daimon-news/pull/582

### CODE REVIEW

File: app/views/categories/show.html.haml [hunk 1]
```diff
@@ -22,5 +22,6 @@
   = render 'banner', position: :press_articles_top
   - if @category.has_children?
     = render 'categories', categories: @category.children, depth: 1
-  - else
-    = render 'posts', posts: @posts
+
+  %h2 このカテゴリの新着記事
```
Comment (tricknotes) [inline app/views/categories/show.html.haml]: Where does this code come from?
I couldn't find in https://github.com/bm-sms/daimon-news/pull/582

Comment (okkez) [inline app/views/categories/show.html.haml]: in my category-design branch.

---

今、ローカルにあるブランチです。デザイン関係は別PRにしようとしてました。

Comment (tricknotes) [inline app/views/categories/show.html.haml]: Please push it to remote repository :-)

Comment (tricknotes) [inline app/views/categories/show.html.haml]: However this message is unnecessary, I think.

**Comment by tricknotes** [comment]
Thanks Review Apps ❤️ 
This feature is landed on master 🚀

**Pull request closed without merging**